packages/crayons-core/src/components/country-phone/country-phone.tsx

```diff
@@ -194,7 +199,11 @@ export class CountryPhone {
   }
 
   componentWillLoad() {
-    this.setPhoneNumberDetails(this.value);
+    if (this.countryCodeDefaultValue) {
+      this.updateAppropriateValue(this.countryCodeDefaultValue);
```

Comment:
@BarathSankar When there's a defaultValue and value, value should take priority over the defaultValue. wdyt?

Reply:
Agreed. Changed it.